content/blog/new-beginnings/index.md

```diff
@@ -1,108 +1,7 @@
 ---
-title: New Beginnings
-date: "2015-05-28T22:40:32.169Z"
-description: This is a custom description for SEO and Open Graph purposes, rather than the default generated excerpt. Simply add a description field to the frontmatter.
+title: New Post!
```

Comment:
Hey @zoeyh88 it looks like you _replaced_ a blog post here, you didn't make a new one. Are you sure you want to do that? 🤔

Reply:
Don't worry @jamsea ! I meant to replace it, I'll change the description of the change so nobody else gets confused! 👍